### PR TITLE
Fluentd proposal

### PR DESCRIPTION
**Pull request by caniszczyk**
RFC for the Fluentd proposal: http://www.fluentd.org/

### CODE REVIEW

**Comment by caniszczyk** [comment]
RFC @cncf/toc

File: proposals/fluentd.adoc [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+*Website*: http://www.fluentd.org/
+
+*Release methodology and mechanics*: The latest release is found at: https://rubygems.org/gems/fluentd
+
+* Generally the last major version and one branch of the old major version are actively supported
+* Changes are included in a minor version if they don’t contain backward incompatibility, otherwise they’re included only in a branch of the next major version
+* The current latest major version is 0.14 and the last major version is 0.12.
+* Releases are created by the Fluentd Core Team as a tag on the git repository and a new release on rubygems.org.
+
+*Social media accounts*: Twitter (https://twitter.com/fluentd) and Facebook (https://www.facebook.com/Fluentd-196064987183037/)
```
Comment (bgrant0607) [inline proposals/fluentd.adoc]: And Slack.
http://slack.fluentd.org/

File: proposals/fluentd.adoc [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+Non Treasure Data committers (with direct write access):
+
+* Naotoshi Seo (https://github.com/sonots)
+* Nobuyuki Kubota (https://github.com/nobu-k)
+* Kosaki (https://github.com/kosaki)
+* Yuichi Tateno (https://github.com/hotchpotch)
+
+*Statement on alignment with CNCF mission*:
+
+Logging is a crucial part of cloud native architectures. Fluentd allows you to unify data collection and consumption for a better use and understanding of data.
```
Comment (bgrant0607) [inline proposals/fluentd.adoc]: http://thenewstack.io/fluentd-offers-comprehensive-log-collection-cloud-microservices-world/

**Comment by caniszczyk** [comment]
final reminder for the @cncf/toc for any comments before an official vote is called

**Comment by bgrant0607** [comment]
LGTM

File: proposals/fluentd.adoc [hunk 3]
```diff
@@ -0,0 +1,112 @@
+== Fluentd Proposal
+
+*Name of project:* Fluentd
+
+*Description:* Fluentd is an open source data collector for unified logging layer. Fluentd's 600+ plugins connect it to many data sources and data outputs. Fluentd has a large adopter community consisting of Atlassian, LINE, Microsoft, Nintendo, GREE (and more).
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: missing an article or similar here - for **a** unified logging layer (or "to facilitate a" perhaps)

File: proposals/fluentd.adoc [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+*Sponsor / Advisor from TOC*: Brian Grant <briangrant@google.com>
+
+*Unique Identifier*: fluentd
+
+*License*: Apache License v2.0
+
+*Source control repositories*:
+
+The donation includes all repositories on the GitHub fluent organization (https://github.com/fluent/):
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: This is a pretty huge list - could we have a brief description of these? (grouping them might make sense, not necessarily one for each)

File: proposals/fluentd.adoc [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+*Infrastructure requirements*: CI and potentially CNCF Community Cluster access
+
+We currently have several unit tests for all components of Fluentd, the tests can be found in the source code here:
+
+https://github.com/fluent/fluentd/tree/master/test
+
+With CNCF infrastructure, we aim to improve our testing with real deployments and critical loads over different connectors that we provide.
+
+*Issue tracker*: https://github.com/fluent/fluentd/issues and respective github issues for each repository.
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: GitHub

File: proposals/fluentd.adoc [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+* Generally the last major version and one branch of the old major version are actively supported
+* Changes are included in a minor version if they don’t contain backward incompatibility, otherwise they’re included only in a branch of the next major version
+* The current latest major version is 0.14 and the last major version is 0.12.
+* Releases are created by the Fluentd Core Team as a tag on the git repository and a new release on rubygems.org.
+
+*Social media accounts*: Slack (http://slack.fluentd.org), Twitter (https://twitter.com/fluentd) and Facebook (https://www.facebook.com/Fluentd-196064987183037/)
+
+*Existing sponsorship*: Treasure Data is a primary sponsor of Fluentd.
+
+*External Dependencies:* The dependencies can be found in the main gemspec for fluentd but are listed below: https://github.com/fluent/fluentd/blob/master/fluentd.gemspec
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: *:

File: proposals/fluentd.adoc [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+Non Treasure Data committers (with direct write access):
+
+* Naotoshi Seo (https://github.com/sonots)
+* Nobuyuki Kubota (https://github.com/nobu-k)
+* Kosaki (https://github.com/kosaki)
+* Yuichi Tateno (https://github.com/hotchpotch)
+
+*Statement on alignment with CNCF mission*:
+
+Logging is a crucial part of cloud native architectures. Fluentd offers comprehensive log collection for microservices and cloud monitoring (http://thenewstack.io/fluentd-offers-comprehensive-log-collection-cloud-microservices-world/).
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: Linking to a New Stack article seems a bit lazy, can we inline a real statement of alignment here?

File: proposals/fluentd.adoc (hunk 3 above)
Comment (jonboulle) [inline proposals/fluentd.adoc]: `GREE (and more).` --> `GREE, and more.`

File: proposals/fluentd.adoc [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+We currently have several unit tests for all components of Fluentd, the tests can be found in the source code here:
+
+https://github.com/fluent/fluentd/tree/master/test
+
+With CNCF infrastructure, we aim to improve our testing with real deployments and critical loads over different connectors that we provide.
+
+*Issue tracker*: https://github.com/fluent/fluentd/issues and respective github issues for each repository.
+
+*Mailing lists:* https://groups.google.com/forum/#!forum/fluentd
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: : snuck inside the :

File: proposals/fluentd.adoc [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+*Website*: http://www.fluentd.org/
+
+*Release methodology and mechanics*: The latest release is found at: https://rubygems.org/gems/fluentd
+
+* Generally the last major version and one branch of the old major version are actively supported
+* Changes are included in a minor version if they don’t contain backward incompatibility, otherwise they’re included only in a branch of the next major version
+* The current latest major version is 0.14 and the last major version is 0.12.
```
Comment (jonboulle) [inline proposals/fluentd.adoc]: so to be clear, fluentd doesn't strictly follow semver? (even though the version numbers suggest so) - can that be tweaked?

File: proposals/fluentd.adoc (hunk 3 above)
Comment (jonboulle) [inline proposals/fluentd.adoc]: It would be nice to talk about community beyond just adopters. Do these companies contribute back?

Comment (edsiper) [inline proposals/fluentd.adoc]: ack.

File: proposals/fluentd.adoc (hunk 4 above)
Comment (edsiper) [inline proposals/fluentd.adoc]: ack.

File: proposals/fluentd.adoc (hunk 5 above)
Comment (edsiper) [inline proposals/fluentd.adoc]: ack

File: proposals/fluentd.adoc (hunk 8 above)
Comment (edsiper) [inline proposals/fluentd.adoc]: ack

File: proposals/fluentd.adoc (hunk 7 above)
Comment (edsiper) [inline proposals/fluentd.adoc]: ack.

**Comment by caniszczyk** [comment]
The official CNCF TOC vote has been called on the mailing list:
https://lists.cncf.io/pipermail/cncf-toc/2016-November/000454.html

**Comment by caniszczyk** [comment]
Friendly reminder to @cncf/toc to vote!

We currently have 4 YES votes from: KenO, BrianG, AlexisR, BenH

**Comment by caniszczyk** [comment]
The Fluentd vote passes, congrats and welcome Fluentd!

+1s from TOC:
Camille Fournier: https://lists.cncf.io/pipermail/cncf-toc/2016-November/000465.html
Ben Hindman: https://lists.cncf.io/pipermail/cncf-toc/2016-November/000462.html
Solomon Hykes: https://lists.cncf.io/pipermail/cncf-toc/2016-November/000464.html
Brian Grant: https://lists.cncf.io/pipermail/cncf-toc/2016-November/000457.html
Alexis Richardson: https://lists.cncf.io/pipermail/cncf-toc/2016-November/000458.html
Ken Owens: https://lists.cncf.io/pipermail/cncf-toc/2016-November/000455.html

We will be working with the Fluentd community over the next few weeks to bring it into the CNCF.

**Pull request merged**